plugins/inputs/proxysql/README.md

```diff
@@ -0,0 +1,52 @@
+# ProxySQL Input Plugin
+
+This plugin gathers the statistic data from ProxySQL server.
+
+* Connection Pool
+* Commands Counters
+* Global Stats
+
+### Configuration
+
+```toml
+[[inputs.proxysql]]
+  ## specify servers via a url matching:
+  ##  [username[:password]@][protocol[(address)]]/]
+  ##  see https://github.com/go-sql-driver/mysql#dsn-data-source-name
+  ##  e.g.
+  ##    servers = ["user:passwd@tcp(127.0.0.1:6032)/"]
+  ##    servers = ["user@tcp(127.0.0.1:6032)/"]
+  #
+  ## If no servers are specified, then localhost is used as the host.
+  servers = ["tcp(127.0.0.1:6032)/"]
```

Comment:
As this is the default please comment this option. Leave only those that are required for the user to fill.
```suggestion
  # servers = ["tcp(127.0.0.1:6032)/"]
```